code/game/objects/structures/janitor.dm

```diff
@@ -80,6 +80,7 @@
 
 /obj/structure/mop_bucket/janitorialcart/Initialize(mapload)
 	. = ..()
+	reagents.maximum_volume *= 2.5
```

Comment:
note that this is a monkestation edit please.

Reply:
> note that this is a monkestation edit please.

It has been done.